mitodl/mitxpro
# #143 Product Page: Who Should Enroll

#### Pre-Flight checklist



#### What are the relevant tickets?
Closes #143 

#### What's this PR do?
Adds "Who Should Enroll" section to Course Page.

#### How should this be manually tested?
Create a "Who should enroll page" type child page on a `CoursePage`, visit the course page and the section should be visible.

#### Where should the reviewer start?
Assuming there are courses in the database and CMS, create a child page as detailed above.

#### Screenshots (if appropriate)
![desktop-who-should-enroll](https://user-images.githubusercontent.com/45350418/56735150-18230800-677e-11e9-8687-6802fdf81576.png)
![mobile-who-should-enroll](https://user-images.githubusercontent.com/45350418/56735163-253ff700-677e-11e9-8ec0-b3fd96fad099.png)
